src/bioregistry/data/bioregistry.json

```diff
@@ -190,6 +193,14 @@
     "example": "T0599",
     "mappings": {
       "cellosaurus": "ABM"
+    },
+    "name": "Applied Biological Materials cell line products",
```

Comment:
This seems to duplicate the name that's already available via cellosaurus, so it can be removed.